.github/workflows/pr-actions.yml

```diff
@@ -48,37 +48,61 @@ jobs:
   # Run Jest Unit Tests
   test-unit:
     uses: ./.github/workflows/run-npm-script.yml
-    needs: lint
+    needs: draft
     with:
       label: 'Test: Unit'
       script: 'npm run test:unit'
 
   # Run Cypress End to End Tests
-  test-e2e:
-    name: 'Test: End to End'
-    runs-on: ubuntu-latest
-    needs: test-unit
-    # stop the job if it runs over 20 minutes to prevent a hanging process from eating minutes
-    timeout-minutes: 45
+  install:
+    needs: draft
```

Comment:
I reworked the sequencing so that the draft check runs first as the only prerequisite, and then lint, unit, and e2e tests run in parallel. 

Before the rationale was to cancel the whole run early when we fail on lint or unit tests, but now that the entire suite completes in less time, I think it's reasonable to start the e2e tests without waiting for linting and unit tests, just to cut down on the wait time